dev-requirements.txt

```diff
@@ -2,5 +2,5 @@ click
 requests
 pytest>=3.5
 pytest-cov
-pytest-flake8
+pytest-flake8==1.0.5
```

Comment:
Why we pinned the pytest-flake8 here?

Reply:
I was debugging why the tests are failing. Seemed that it jumped between previous CI run, where Flake now gave an error on an untouched file:
```
/home/travis/build/lord63/choosealicense-cli/choosealicense/test/responses/get_json.py:23:38: E741 ambiguous variable name 'l'
```

Reply:
But pinning it made the CI explode for some reason :/

Reply:
Don't worry, I'll merge it first and take a look at it. 👍